appveyor.yml

```diff
@@ -5,11 +5,32 @@ configuration:
 platform:
 - Win32
 
+environment:
+  matrix:
+  - BUILD_SYSTEM: MSVC
+  - BUILD_SYSTEM: CMAKE
+
+matrix:
+  allow_failures:
+  - BUILD_SYSTEM: CMAKE
```

Comment:
If we're going to have this in tree, I'd rather it not be optional. It needs to either be maintained or removed.